### PR TITLE
make main page slightly less ugly

### PR DESCRIPTION
**Pull request by ozyx**
just some general formatting. still a lot of work to be done here.

### CODE REVIEW

File: server/index.html [hunk 1]
```diff
@@ -3,51 +3,51 @@
...
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
     <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
-    <script>
+    <!--<script>
         setTimeout(function(){
             window.location = 'https://www.youtube.com/watch?v=dQw4w9WgXcQ';
          }, 3000);
-    </script>   
-	</body>
+    </script>   -->
```
Comment (reddraggone9) [inline server/index.html]: Why not just remove it outright?

Comment (ozyx) [reply server/index.html]: removed.

**Review by fel1xw — APPROVED** [review]

**Comment by ozyx** [comment]
Does @chaosbot remove `conflicts` label once conflicts are resolved? If not an issue should be created.

File: server/index.html [hunk 2]
```diff
@@ -17,25 +18,31 @@
...
-  </a>
-  <div class="container">
-      <div class="content">
-          <p>Hi, this is <strong>ChaosBot</strong>.</p>
-          <p>For more information see <a href="https://github.com/chaosbot/chaos">Github</a>!</p>
-      </div>
-  </div>
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-  </body>
+<body onload="gameLoop()" onkeydown="keyDown(event)" onkeyup="keyUp(event)">
```
Comment (rhengles) [inline server/index.html:24]: Lies

**Comment by ozyx** [comment]
@rhengles elaborate? Did I bork the merge?

**Comment by rhengles** [comment]
Well, you're reintroducing the chicken game, it's not just "some general formatting".

**Comment by ozyx** [comment]
So I messed up merging. Sorry, I haven't paid much attention to the merges today. I will fix this later tonight. It was not my intention to reintroduce something which was removed.

**Comment by ozyx** [comment]
¯\\_(ツ)_/¯
